x-pack/plugins/siem/public/components/formatted_duration/tooltip/index.tsx

```diff
@@ -22,7 +23,11 @@ export const FormattedDurationTooltipContent = pure<{
   <>
     {tooltipTitle != null ? <P data-test-subj="title">{tooltipTitle}</P> : null}
     <P data-test-subj="humanized">{getHumanizedDuration(maybeDurationNanoseconds)}</P>
-    <P data-test-subj="raw-value">raw: {maybeDurationNanoseconds}</P>
+    <P data-test-subj="raw-value">
+      <FormattedMessage id="xpack.siem.formattedDuration.tooltipLabel" defaultMessage="raw" />
+      {': '}
+      {maybeDurationNanoseconds}
```

Comment:
This can be combined into a string interpolation and look better.

Reply:
Since this can possibly be an object, I opted to not make the above change so I do not get a `[object object]` string displaying on the DOM.

Reply:
My question is probably nothing to do what the PR intends to, but wondering if it is possibly be an object, like what I'm imaging:
```
{ 123: { 456: 789 } }
```
Do we need to handle this case specifically? Or this is not the object that I'm thinking.
If so, how would you suggest when we use object as a type rather than specify the detail of it?